### ISSUE TITLE
Suggestion: standalone run for remove-duplicate-uuids

### ISSUE THREAD

**Issue by Endag** [maruohon/worldutils#4]
One major downside for remove-duplicate-uuids command is necessity for chunks to be unloaded to be processed. Therefore this command cannot be run automatically on periodic basis as it needed to remove all chunkloading before run.

So I have a question: is it possible to make this algo to be running outside of minecraft server? If it can be done, it will be possible to insert this fix in servers restart procedure for example, i.e. : stop the server, run duplicate clean on world folder, restart server.

**Comment by maruohon** [comment]
Yes that's certainly possible. I'm (very slowly) working on another project (a backup tool and NBT tool of sorts, written in C at least for now), where this feature would fit in and could be added to. But that project was just on a 4-year hiatus since the previous commit... so there is no telling how long it will take until it actually gets into a usable state.

So maybe a more realistic option (or at least a lot faster to implement) might be to build this specific feature either into the mod-jar as a standalone runnable feature (if that's possible...), or copy & paste most of the code and make it as a separate program in Java. I'll need to investigate those options when I get the time.

There *might* also be another option actually. If there would be a blocking version of the command (instead of the current scheduler system), then a mod like World Primer could be used to run the command in the server start-up phase, before the world has loaded.
